package com.example.exception;

public class Main {

	
	public static void main(String[] args) {
		
		test();
		
		/*
		 * try { UseOfException useOfException = new UseOfException();
		 * useOfException.returnStatus("Satya", 100); System.out.println("Main Class");
		 * } catch (Exception e) { System.out.println("Exception in main class"); //
		 * TODO: handle exception e.printStackTrace(); }
		 */
		
	
		
		
	}
	
	
	public static int test() {
		return test();
		
	}
}
